src/api/app/views/webui/users/tokens/edit.html.haml

```diff
@@ -22,7 +22,7 @@
                 = @token.id
               .form-group
                 = f.label(:type, 'Operation:')
-                = @token.class.token_name.capitalize
+                = @token.capitalize
```

Comment:
Here as well

Reply:
the same thing: We are talking about the operation.. it is correct.

Reply:
Same here:

```suggestion
                = @token.to_s.capitalize
```